.github/workflows/image.yml

```diff
@@ -0,0 +1,134 @@
+name: 'image'
+
+on:
+  pull_request:
```

Comment:
image build will work even a pull_request is created?

Reply:
yes, only build check without push, see the ci results in current PR.